src/controllers/stallController.ts

```diff
@@ -215,9 +207,9 @@ async function destroyStallImages(req: Request, res: Response, next: NextFunctio
 }
 
 export const findStallsByIdsFunc = findStallsByIds;
-export const mapStallToCardFunc = mapStallToCard;
+export const findAllStallsFunc = indexStall;
```

Comment:
Erm whats the diff between `findAllStallsFunc` and `indexStallFuncs`? Don't both of these functions return all stalls?